docs/reference-implementation/Managed-prometheus.md

```diff
@@ -0,0 +1,43 @@
+# Managed prometheus in Azure Mission-critical (preview)
+
+The Azure Mission-Critical reference implementation implements a wide variety of monitoring and observability capabilities and best practices. It uses regional and global Log Analytics workspaces to store logs and metrics, decoupled from the ephemeral nature and lifecycle of it's stamps, it contains an instrumented sample application workload using Application Insights for in-depth application metrics and distributed traces, uses a health service to evaluate and report the status of each of its deployed stamps and uses a self-hosted instance of Grafana to visualize the user and system flows and provide actionable insights into the overall health status of the workload. This is a good start, but might not be detailed enough for production workloads.
+
+To further improve the observability capabilities of a mission-critical workload build based on top of a Azure Mission-Critical reference implementation, Prometheus can be used to gain more granular metrics from the infrastructure as well as from the workload.
+
+> **Prometheus** is an open-source monitoring solution that enables you to collect metrics from various systems, applications, and services. It allows you to query and visualize your data, alert on critical conditions, and understand the performance and behavior of your infrastructure. It is built with a focus on scalability, reliability, and simplicity, making it a popular choice for monitoring large and complex systems.
+
+## Why Prometheus?
+
+Azure Mission-Critical follows a cloud-native-first, PaaS-first approach. Whenever possible should PaaS and cloud-native services be prioritized when designing, developing or modernizing a workload on Azure. These cloud-native, and several PaaS services have one thing in common, they provide prometheus metrics. To call out a few examples, AKS is using Kubernetes, Kubernetes provides Prometheus metrics on various levels like the control plane and the nodes (kubelet) but also its services like cert-manager, open service mesh and nginx (and of course many more) provide prometheus metric endpoints.
+
+As you can see, Prometheus, Prometheus metrics and especially the Prometheus Query Language (PromQL) are a defacto standard in the cloud native space. What options do we have to leverage these metrics? The existing Azure Monitor agent already supports scraping prometheus metrics and store them in Log Analytics. This was already quiet helpful, but, Log Analytics is a quiet expensive storage and it supports Kusto Query Language (KQL) but not the Prometheus Query Language (PromQL). And this can become a problem, especially when using Prometheus across environments and platforms.
```

Comment:
Instead of de facto standard should we call out that Prometheus an official cloud native standard (CNCF)?

Reply:
I cannot even remember that I wrote this text. That old is this PR already..

```suggestion
As the examples above show, Prometheus, Prometheus metrics and especially the Prometheus Query Language (PromQL) are well-established standards in the cloud native space. The existing Azure Monitor add-on for Azure Kubernetes Service already supported scraping prometheus metrics from endpoints within a cluster and stored them in Log Analytics. Azure Monitor Workspace now supports also PromQL which makes it easier to use across environments and platforms.
```